### PR TITLE
update waas-restart.md

### PR DESCRIPTION
**Pull request by MaratMussabekov**
issue #781 ,
added notification about behavior related to disconnected RDP sessions

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/deployment/update/waas-restart.md [hunk 1]
```diff
@@ -42,6 +42,9 @@ When **Configure Automatic Updates** is enabled in Group Policy, you can enable
 - **Turn off auto-restart for updates during active hours** prevents automatic restart during active hours.
 - **No auto-restart with logged on users for scheduled automatic updates installations** prevents automatic restart when a user is signed in. If a user schedules the restart in the update notification, the device will restart at the time the user specifies even if a user is signed in at the time. This policy only applies when **Configure Automatic Updates** is set to option **4-Auto download and schedule the install**.
 
+>[!NOTE]
```
Comment (nenonix) [inline windows/deployment/update/waas-restart.md]:
```suggestion
> [!NOTE]
```

File: windows/deployment/update/waas-restart.md [hunk 2]
```diff
@@ -42,6 +42,9 @@ When **Configure Automatic Updates** is enabled in Group Policy, you can enable
...
+>In case of using Remote Desktop Protocol connections, only active RDP sessions are considered as logged on users. Devices, that do not have locally logged on users or active RDP sessions, will be restarted.
```
Comment (nenonix) [inline windows/deployment/update/waas-restart.md]:
```suggestion
>When using Remote Desktop Protocol connections, only active RDP sessions are considered as logged on users. Devices, that do not have locally logged on users, or active RDP sessions, will be restarted. 
```

File: windows/deployment/update/waas-restart.md [hunk 3]
```diff
@@ -159,8 +162,9 @@ In the Group Policy editor, you will see a number of policy settings that pertai
 
 >[!NOTE]
 >You can only choose one path for restart behavior.
->
 >If you set conflicting restart policies, the actual restart behavior may not be what you expected.
+>In case of using RDP, only active RDP sessions are considered as logged on users.
```
Comment (nenonix) [inline windows/deployment/update/waas-restart.md]:
```suggestion
>When using RDP, only active RDP sessions are considered as logged on users.
```

File: windows/deployment/update/waas-restart.md (hunk 2 above)
Comment (JohanFreelancer9) [inline windows/deployment/update/waas-restart.md]:
```suggestion
> When using Remote Desktop Protocol connections, only active RDP sessions are considered as logged on users. Devices that do not have locally logged on users, or active RDP sessions, will be restarted. 

```

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9 , @nenonix 
your recommendations were applied, thank you!

**Comment by mypil** [comment]
@jaimeo @DaniHalfin - Can you please assist to check, approve and merge the content for this PR based on issue #781?

Thank you.

**Pull request merged**